CHANGELOG.md

```diff
@@ -0,0 +1,18 @@
+- Adds `functions.logger` SDK to enabled structured logging in the Node.js 10 runtime. Example:
+
+  ```js
```

Comment:
Wow, does this work?  In the d.ts files we use ```javascript, but a shorter form would be nice.

Reply:
Yup 😄